<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>LevaConnectionWindow</class>
 <widget class="QMainWindow" name="LevaConnectionWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>775</width>
    <height>436</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>SQL</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QLabel" name="label">
        <property name="text">
         <string>Server Name:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="txtServerName"/>
      </item>
      <item>
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>User Name:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="txtUserName"/>
      </item>
      <item>
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string>Password:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="txtPassword">
        <property name="echoMode">
         <enum>QLineEdit::Password</enum>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_4">
        <property name="text">
         <string>Port:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QSpinBox" name="spinBox">
        <property name="maximum">
         <number>99999</number>
        </property>
        <property name="value">
         <number>3306</number>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="cmdConnect">
        <property name="text">
         <string>Connect</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QSplitter" name="splitter_2">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <widget class="QTreeWidget" name="treeWidget">
       <property name="contextMenuPolicy">
        <enum>Qt::CustomContextMenu</enum>
       </property>
       <property name="headerHidden">
        <bool>true</bool>
       </property>
       <column>
        <property name="text">
         <string notr="true">1</string>
        </property>
       </column>
      </widget>
      <widget class="QSplitter" name="splitter">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <widget class="QTextEdit" name="txtQuery"/>
       <widget class="CustomQueryView" name="trvData"/>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>775</width>
     <height>25</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuDatabase">
    <property name="title">
     <string>Database</string>
    </property>
    <widget class="QMenu" name="menuSet_Current_Db">
     <property name="title">
      <string>Set Current Db</string>
     </property>
     <addaction name="separator"/>
    </widget>
    <addaction name="actionAdd_New"/>
    <addaction name="menuSet_Current_Db"/>
   </widget>
   <widget class="QMenu" name="menuUsers">
    <property name="title">
     <string>Users</string>
    </property>
    <addaction name="actionUser_Manager"/>
   </widget>
   <widget class="QMenu" name="menuQuery">
    <property name="title">
     <string>Query</string>
    </property>
    <addaction name="actionExecute"/>
   </widget>
   <addaction name="menuDatabase"/>
   <addaction name="menuUsers"/>
   <addaction name="menuQuery"/>
  </widget>
  <action name="actionAdd_New">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/images/Icon_193.ico</normaloff>:/images/Icon_193.ico</iconset>
   </property>
   <property name="text">
    <string>Add New</string>
   </property>
  </action>
  <action name="actionUser_Manager">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/images/Icon_174.ico</normaloff>:/images/Icon_174.ico</iconset>
   </property>
   <property name="text">
    <string>User Manager</string>
   </property>
  </action>
  <action name="actionExecute">
   <property name="icon">
    <iconset resource="../../MegvelERP/src/res.qrc">
     <normaloff>:/images/base_cog_32.png</normaloff>:/images/base_cog_32.png</iconset>
   </property>
   <property name="text">
    <string>Execute</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+R</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>CustomQueryView</class>
   <extends>QTreeView</extends>
   <header>customqueryview.h</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="res.qrc"/>
  <include location="../../MegvelERP/src/res.qrc"/>
 </resources>
 <connections/>
</ui>
